common/src/main/java/org/mcaccess/minecraftaccess/utils/condition/IntervalKeystroke.java

```diff
@@ -46,4 +46,13 @@ public boolean canBeTriggered() {
     protected boolean otherTriggerConditions() {
         return true;
     }
+
+    @Override
+    public void updateStateForNextTick() {
```

Comment:
Wait, I haven't saw the usage of it... can you provide the commit that contains this change? I want to know the purpose.
Wait, is that left fixing? In the access menu shortcuts bar PR?

Reply:
IntervalKeystroke had a problem where if you double tap a key within the interval it will just ignore it which is not good. This fixes it by making the interval ready on key down which means it will always execute no matter how quickly you tap the key. A quick Git blame or scrolling through the commit log would tell you that this change was made in commit 14a284bd926c255011f556bf05fd7c2f1f0a2fff.

Reply:
Sorry... I'm on my phone, does the logic same as the fixing I made for the... previous problem?

Reply:
Back to the PC now, IIRC it's: https://github.com/minecraft-access/minecraft-access/pull/349#issuecomment-2480888163